CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 - **Breaking Change: Rename `bin/newrelic` command to `bin/newrelic_rpm`**
   The executable file for the agent's CLI has been renamed from `bin/newrelic` to `bin/newrelic_rpm`. This change resolves a name collision with the standalone New Relic CLI tool. [PR#3323](https://github.com/newrelic/newrelic-ruby-agent/pull/3323)
 
+- **Feature: Add argument validation for the `Agent#record_custom_event` API**
```

Comment:
I think customers have to put `NewRelic::` in front of agent for the API call to work.

```suggestion
- **Feature: Add argument validation for the `NewRelic::Agent#record_custom_event` API**
```